ansible_collections/arista/avd/docs/contribution/style-guide.md

```diff
@@ -1,12 +1,26 @@
 # AVD Coding Style
 
-This page provides a list of guidelines to apply when developing the Jinja2 template in the AVD context. These rules apply for either creation or update of any Jinja2 file available in a `aristanetworks/ansible-avd` repository.
+This page provides a list of guidelines to apply when developing Python code or Jinja2 template in the AVD context. These rules apply for either creation or update of any Python or Jinja2 file available in a `aristanetworks/ansible-avd` repository.
 
 ## Python code style
 
 As AVD is an Ansible collection, we're required to follow guidelines from the official [Ansible documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_best_practices.html) for all Python code.
```

Comment:
Let's state clearly here that we expect the line-length to be 160 to make it clear for people looking at this

Reply:
ack